src/style.css

```
@@ -0,0 +1,38 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-around;
+  height: 16px;
+  width: 100px;
+}
+
+.stars_star {
+  background: url("./images/star.svg") no-repeat;
+  width: 16px;
+  margin: 2px 0 0;
+}
+
+.stars--1 :first-child {
+  background: url("./images/star-active.svg") no-repeat;
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background: url("./images/star-active.svg") no-repeat;
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background: url("./images/star-active.svg") no-repeat;
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background: url("./images/star-active.svg") no-repeat;
+}
+
+.stars--5 :nth-child(n) {
+  background: url("./images/star-active.svg") no-repeat;
+}
```

```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(n) {
  background: url("./images/star-active.svg") no-repeat;
}
```
Не потрібно дублювати стилі. Використай комбінований селектор